.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
-/dist
\ No newline at end of file
+/dist
+/.vscode
```

Comment:
ooh I have opinions on this.
Absolutely fine for now, but trouble here is that loads of folk's own preferences can pile up in this file if we're not careful.

You can get mac, linux windows, intellj all sorts of ignores that are not relevant for you.

But there's a better way. You can also create a global gitignore file that'll ignore across every repo on your device:
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Feel free to leave it for now, just FYI!

Reply:
Didn't know this, but that sounds great! Will implement a global .gitignore separately.